pkg/test/scopes/scopes.go

```diff
@@ -19,7 +19,4 @@ import "istio.io/pkg/log"
 var (
 	// Framework is the general logging scope for the framework.
 	Framework = log.RegisterScope("tf", "General scope for the test framework", 0)
-
-	// CI system specific logging scope.
-	CI = log.RegisterScope("ci", "Scope for normal log reporting to be used in CI systems", 0)
```

Comment:
What do you think about "Framework" as a name for the variable? Would "Testing" be more clear?

Reply:
I am open to name changes.  I have this item added to the test and release work group meeting today so I can get more feedback.